src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (number) => {
+    if (!number && number !== 0) {
```

Comment:
Could`ve used more simple check.
```suggestion
    if (number === undefined) {
```